activejob/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Deprecate `sidekiq` as an adapter option.
```

Comment:
I think we need to be very intentional with the wording here. To me this currently sounds like `sidekiq` is deprecated completely, when actually its just "the built in sidekiq adapter".